packages/slate-editor/src/modules/editor/menuOptions.tsx

```diff
@@ -274,6 +275,18 @@ function* generateOptions(
         };
     }
 
+    if (withEmbeds && withSpecificProviderOptions) {
+        yield {
+            action: MenuAction.ADD_DROPBOX,
+            icon: Icons.ComponentDropbox,
+            group: Group.SPECIFIC_EMBEDS,
```

Comment:
I will address the new group hierarchy at the end (once all new menu options are added)

There is [DEV-11512](https://linear.app/prezly/issue/DEV-11512/rearrange-the-groups-to-match-the-new-categories) to address that.